docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_extended_security_updates/_index.md

```diff
@@ -0,0 +1,173 @@
+    - _`both`_: to deploy both a Windows Server 2012 VM and a SQL Server 2012 VM that will be Arc enabled.
+
+  ![Parameters file](./01.png)
+
+- To run the automation, navigate to the [deployment folder](https://github.com/microsoft/azure_arc/tree/main/azure_arc_servers_jumpstart/esu/bicep). From the deployment folder run the below command:
+
+  ```shell
+    az group create --name <your_resource_group> --location <your_azure_region>
+    az deployment group create --resource-group <your_resource_group> main.bicep --parameters main.parameters.json
+  ```
+
+    For example:
+
+  ```shell
+    az group create --name Arc-ESU-Demo --location "westeurope"
+    az deployment group create --resource-group Arc-ESU-Demo --name Arc-ESU-Demo --template-file main.bicep --parameters main.parameters.jsonn
+  ```
+
+- Once the automation finishes its run, you should see an output as follows:
+
+  ![Automation output](./02.png)
+
+ > **NOTE: For the script to work properly you must run this command from the deployment folder.**
+
+- After the script has finished its run verify the resources are created on the Azure Portal:
+
+    ![Resources created on resource group](./03.png)
+
+    ![Resources created on resource group](./04.png)
+
+## Windows Login & Post Deployment
+
+- Now that the Azure Windows Server VM is created, it is time to connect to it using either RDP or Azure Bastion.
+
+    ![Azure Bastion session 01](./05.png)
+
+    ![Azure Bastion session 02](./06.png)
+
+    ![Azure Bastion session 03](./07.png)
+
+- At first login, as mentioned in the "Automation Flow" section, a logon script will get executed. This script was created as part of the automated deployment process.
+
+    ![Script's output](./08.png)
+
+    ![Script's output](./09.png)
+
+    ![Script's output](./10.png)
+
+    ![Script's output](./11.png)
+
+- Let the script to run its course and **do not close** the Powershell session, this will be done for you once completed.
+
+    > **NOTE: The script run time is ~10-15 min long.**
+
+- Upon successful run, a new Azure Arc-enabled server and/or Azure Arc-enabled SQL server will be added to the resource group.
+
+  ![Screenshot Azure Arc-enabled resources on resource group](./12.png)
+
+- Now that you have successfully onboarded the Arc-enabled resources, you will be able to manage the Extended Security Updates (ESU) from the Azure Portal:
+
+## Extended Security Updates (ESU)
+
+Now that you have Windows Server 2012 and/or SQL Server 2012 R2 Arc-enabled, you are able to manage ESU licenses for these servers.
+
+- Navigate to the Azure Arc page
+
+  ![Screenshot Azure Arc navigate](./13.png)
+
+- Select Extended Security Updates in the left pane.
+
+  ![Screenshot Azure Arc Extended](./14.png)
+
+- Provision Windows Server 2012 and 2012 R2 Extended Security Update licenses from Azure Arc.
+
+  ![Screenshot ESU Licenses](./15.png)
+
+  ![Screenshot ESU Licenses](./16.png)
```

Comment:
The screenshot is missing the resource group selection and # of cores. Any specific reason?